nucleus/admin/cli/src/main/java/com/sun/enterprise/admin/cli/CLICommand.java

```diff
@@ -1429,7 +1429,8 @@ private static void file2Set(String file, Set<String> set) {
     protected void buildTerminal() {
         try {
             if (terminal == null) {
-            terminal = TerminalBuilder.builder()
+                System.setProperty("org.jline.terminal.exec.redirectPipeCreationMode", "native");
```

Comment:
Would be better if we referenced the static strings

```suggestion
                System.setProperty(TerminalBuilder.PROP_REDIRECT_PIPE_CREATION_MODE, TerminalBuilder.PROP_REDIRECT_PIPE_CREATION_MODE_NATIVE);
```